test/support/fixtures/keywords_fixtures.ex

```diff
@@ -0,0 +1,44 @@
+defmodule ElixirSearchExtractor.KeywordsFixtures do
+  def valid_file_name, do: "Test File"
+
+  def valid_csv_file,
+    do: %Plug.Upload{
+      content_type: "text/csv",
+      path: "#{File.cwd!()}/test/support/fixtures/csv_files/valid_keywords.csv",
+      filename: "valid.csv"
+    }
+
+  def large_csv_file,
+    do: %Plug.Upload{
+      content_type: "text/csv",
+      path: "#{File.cwd!()}/test/support/fixtures/csv_files/large_file.csv",
+      filename: "large.csv"
+    }
+
+  def invalid_extension_file,
+    do: %Plug.Upload{
+      content_type: "text/csv",
+      path: "#{File.cwd!()}/test/support/fixtures/csv_files/invalid_extension.png",
+      filename: "invalid.png"
+    }
+
+  def valid_keyword_file_attributes(attrs \\ %{}) do
+    Enum.into(attrs, %{
+      "name" => valid_file_name(),
+      "csv_file" => valid_csv_file()
+    })
+  end
+
+  def keyword_file_fixture(user_id, attrs \\ %{}) do
+    {:ok, keyword_file} =
+      attrs
+      |> valid_keyword_file_attributes()
+      |> ElixirSearchExtractor.FileUpload.create_keyword_file(user_id)
```

Comment:
I'm not quite sure about upload the csv file here, l think it might be possible for us to forget to remove the file later 🤔 

Imagine if we are going to test the keywords list, we might only want to create the keyword file records in db not the upload file and assume that file is already uploaded 💡

Reply:
@rosle I have added factories for this purpose - https://github.com/rafayet-monon/elixir_search_extractor/pull/35/files#diff-c572490954dc634bd7da5951e63e3f2b79c306dd5e22a0f41c25ca2c301b484dR1